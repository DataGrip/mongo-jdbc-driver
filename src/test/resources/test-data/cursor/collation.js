// before
db.col.insertOne({"_id": 1, "x": "a"});
db.col.insertOne({"_id": 2, "x": "A"});
db.col.insertOne({"_id": 3, "x": "รก"});
// command
db.col.find({x: "a"}).collation({locale: "en_US", strength: 1});
// clear
db.col.drop();