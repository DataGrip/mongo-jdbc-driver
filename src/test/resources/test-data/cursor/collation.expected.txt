|map|
|Document{{_id=1, x=a}}|
|Document{{_id=2, x=A}}|
|Document{{_id=3, x=รก}}|